nodes/core/core/75-exec.js

```diff
@@ -46,10 +46,11 @@ module.exports = function(RED) {
                 var arg = node.cmd;
                 if ((node.addpay === true) && msg.hasOwnProperty("payload")) { arg += " "+msg.payload; }
                 if (node.append.trim() !== "") { arg += " "+node.append; }
-                // slice whole line by spaces (trying to honour quotes);
-                arg = arg.match(/(?:[^\s"]+|"[^"]*")+/g);
+                // slice whole line by spaces and removes any quotes since spawn can't handle them
+                arg = arg.match(/(?:[^\s"]+|"[^"]*")+/g).map((a) => {
+                  if (/^".*"$/.test(a)) { return a.slice(1,-1)} else {return a};
+                });
                 var cmd = arg.shift();
-                if (/^".*"$/.test(cmd)) { cmd = cmd.slice(1,-1); }
```

Comment:
Hi! I think that would remove characters that you would want to keep in the case where you don't insert any quotes at all. We want to search for quotes first right? And then remove them.